nuvlaedge/agent/orchestrator/kubernetes.py

```diff
@@ -500,7 +523,8 @@ def _to_k8s_obj_name(name: str) -> str:
     def _container_def(image, name,
                        volume_mounts: List[client.V1VolumeMount] | None,
                        command: str = None,
-                       args: str = None) -> client.V1Container:
+                       image_pull_policy: str = None,
```

Comment:
What happen if it is `None` ? 
Does it use a default ? If yes, what is the default and where does it come from (Kubernetes Python library, Kubernetes "daemon" configuration/default ?

Except if it uses a default (on the daemon) based on user configuration, I would specify a default here.

Reply:
I have added the default using the DEFAULT constant at the top of the class.